zipline/data/loader.py

```diff
@@ -154,7 +150,7 @@ def get_benchmark_filename(symbol):
     return "%s_benchmark.csv" % symbol
 
 
-def load_market_data(bm_symbol='^GSPC'):
+def load_market_data(trading_day, trading_days, bm_symbol='^GSPC'):
```

Comment:
Let's add default values for trading_day and trading_days (the values from zipline.utils.tradingcalendar) to support backwards-compatability.